cssbeautify/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for CSS Beautify v0.3.1
+// Project: https://github.com/senchalabs/cssbeautify
+// Definitions by: rictic <https://github.com/rictic>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+  /**
+   * A string used for the indentation of the declaration (default is 4
+   * spaces).
+   */
+  indent?: string;
+  /**
+   * Defines the placement of open curly brace, either end-of-line (default)
+   * or separate-line
+   */
+  openbrace?: 'end-of-line'|'separate-line';
+
+  /**
+   * Always inserts a semicolon after the last ruleset(default is false)
+   */
+  autosemicolon?: boolean;
+}
+declare namespace beautify {}
```

Comment:
I see, that makes sense. Better to match es module semantics for es imports and clearly delineate nonstandard imports.

Is there an issue or wiki page on the Typescript repo or similar with more info? I think my team is going to want to do some evaluation of our options, as we'll likely want to migrate our code and dependencies along these lines.

Thanks for the informative reviews by the way!